core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceMethod.java

```diff
@@ -1,712 +0,0 @@
-import com.linecorp.armeria.common.RequestContext;
-import com.linecorp.armeria.common.util.Exceptions;
-import com.linecorp.armeria.common.util.SafeCloseable;
-import com.linecorp.armeria.internal.AnnotatedHttpServiceParamUtil;
-import com.linecorp.armeria.internal.AnnotatedHttpServiceParamUtil.EnumConverter;
-import com.linecorp.armeria.internal.FallthroughException;
-import com.linecorp.armeria.server.annotation.BeanRequestConverterFunction;
-import com.linecorp.armeria.server.annotation.Default;
-import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
-import com.linecorp.armeria.server.annotation.Param;
-import com.linecorp.armeria.server.annotation.RequestConverterFunction;
-import com.linecorp.armeria.server.annotation.RequestObject;
-import com.linecorp.armeria.server.annotation.ResponseConverterFunction;
-
-import io.netty.util.AsciiString;
-
-/**
- * Invokes an individual method of an annotated service. An annotated service method whose return type is not
- * {@link CompletionStage} or {@link HttpResponse} will be run in the blocking task executor.
- */
-final class AnnotatedHttpServiceMethod {
-    private static final Logger logger = LoggerFactory.getLogger(AnnotatedHttpServiceMethod.class);
-
-    private final Object object;
-    private final Method method;
-    private final List<Parameter> parameters;
-    private final boolean isAsynchronous;
-    private final AggregationStrategy aggregationStrategy;
-    private final List<ExceptionHandlerFunction> exceptionHandlers;
-    private final List<RequestConverterFunction> requestConverters;
-    private final List<ResponseConverterFunction> responseConverters;
-
-    AnnotatedHttpServiceMethod(Object object, Method method, PathMapping pathMapping,
-                               List<ExceptionHandlerFunction> exceptionHandlers,
-                               List<RequestConverterFunction> requestConverters,
-                               List<ResponseConverterFunction> responseConverters) {
-        this.object = requireNonNull(object, "object");
-        this.method = requireNonNull(method, "method");
-        requireNonNull(pathMapping, "pathMapping");
-        this.exceptionHandlers = ImmutableList.copyOf(
-                requireNonNull(exceptionHandlers, "exceptionHandlers"));
-        this.requestConverters = ImmutableList.copyOf(
-                requireNonNull(requestConverters, "requestConverters"));
-        this.responseConverters = ImmutableList.copyOf(
-                requireNonNull(responseConverters, "responseConverters"));
-
-        parameters = parameters(method, pathMapping.paramNames(), !requestConverters.isEmpty());
-        final Class<?> returnType = method.getReturnType();
-        isAsynchronous = HttpResponse.class.isAssignableFrom(returnType) ||
-                         CompletionStage.class.isAssignableFrom(returnType);
-        aggregationStrategy = AggregationStrategy.resolve(parameters);
-
-        this.method.setAccessible(true);
```

Comment:
I found the problem of CI, so we need to call this in new code.

Reply:
Maybe need to add test for `IllegalAccess` case.

Reply:
Thanks, @kojilin ! I missed that code. 😱